script.js

```diff
@@ -1,12 +1,208 @@
+          <span class="badge text-bg-dark">Runtime: ${show.runtime}</span>
+          <p>${show.summary}</p>
+          </div>
+        </div>
+      </div>
+    `;
+  });
+  showEpisodseRow.innerHTML = output;
+
+  // Create option for each show in select after loading page
+  allShows.forEach((show) => {
+    const option = document.createElement("option");
+    option.value = show.name;
+    option.textContent = show.name;
+    selectShows.appendChild(option);
+  });
+}
+
+// Search specific Shows based on select
+async function selectShow(event) {
+  if (event.target.value == "allSerials") {
+    setup();
+    //Remove options from episodes's select
+    const options = selectEpisodes.getElementsByTagName("option");
+    for (let i = options.length - 1; i > 0; i--) {
+      selectEpisodes.removeChild(options[i]);
+    }
+    searchUser.value = "";
+    searchUser.disabled = true;
+  } else {
+    searchUser.value = "";
+    searchUser.disabled = false;
+    //Remove options from episodes's select
+    const options = selectEpisodes.getElementsByTagName("option");
+    for (let i = options.length - 1; i > 0; i--) {
+      selectEpisodes.removeChild(options[i]);
+    }
+    // Find selected movie
+    const specificShow = allShows.find(
+      (show) => show.name === event.target.value
+    );
+    // Determine show's id
+    const specificShowID = specificShow.id;
+    // Get all show's episodes
+    allEpisodes = await getAllAvailableEpisodes(specificShowID);
+    // make card for each episode
+    makeCardForEpisodes(allEpisodes);
+  }
 }
 
-window.onload = setup;
+//Get all episodes via fetch
+async function getAllAvailableEpisodes(showId) {
+  // await response of the fetch call
+  const response = await fetch(
+    `https://api.tvmaze.com/shows/${showId}/episodes`
+  );
+  // only proceed once its resolved
+  const data = await response.json();
+  // only proceed once second promise is resolved
+  return data;
+}
+
+//make card for each episode
+function makeCardForEpisodes(episodeList) {
+  let output = "";
+  episodeList.forEach((episode) => {
+    output += `
+      <div class="col">
+        <div class="card h-100 px-0 mx-1">
+        <img src="${episode.image.medium}" class="card-img-top mx-0" alt="...">
+        <div class="card-body mb-4">
+        <h5 >${episode.name} - S${episode.season
+      .toString()
+      .padStart(2, "0")}E${episode.season.toString().padStart(2, "0")}</h5>
+      <p>${episode.summary}</p>
+        </div>
+        <a href="${
+          episode.url
+        }" class="btn btn-primary" style="position:absolute; bottom:10px; left:20px; right:20px;" target="_blank">See More</a>
+      </div>
+      </div>
+    `;
+  });
+  showEpisodseRow.innerHTML = output;
+
+  // Create option for each episodes in select after loading page
+  episodeList.forEach((episode) => {
+    const option = document.createElement("option");
+    option.value = `S${episode.season
```

Comment:
As above, I'd extract more named variables here, but also, because we're doing this multiple times, I'd extract a _function_ to take an episode and return an identifier. So that here you could just write:
```js
option.value = makeEpisodeIdentifier(episode);
```

Partially this is so that if we change how we make an identifier, we only have to do so in one place. Partially it's to make the code easier to read because we're communicating our _intent_ (what we want to do, rather than how we're doing it) in the function name.

But also, you use this value in both the value and the textContent of the option, so I'd extract a variable for it too:

```js
let episodeIdentifier = makeEpisodeIdentifier(episode);
option.value = episodeIdentifier;
option.textContent = `${episodeIdentifier} - ${episode.name}`;
```